macro-mathjax-ui/src/main/resources/Macros/MathJaxMacro.xml

```diff
@@ -187,6 +174,49 @@ y = \frac{c}{cb-ad}
       load: ['[tex]/noerrors']
     }
   };
+
+  /**
+   * Reset the tex labels, since re-typesetting might create duplicate labels, and perform an asynchronous typesetting.
+   * See https://docs.mathjax.org/en/latest/web/typeset.html#handling-asynchronous-typesetting.
+   */
+  var clearAndTypesetPromise = function(elems) {
+    let typesetPromise = Promise.resolve();
```

Comment:
The initialization doesn't seem to be needed because you're overwriting the value on both branches of the IF.